src/com/dakl/Main.java

```diff
@@ -91,13 +95,15 @@ static void threeByThree(Team center_team, Team[] surrounding, Space center, boo
     public static boolean inputIsValid(CheckerBoard game, String input) {
         String[] command = input.split(" ");
 
+        // REVIEW : Variable java naming convention
         Space unvalidated_pc_loc = null;
         Space unvalidated_end_space = null;
 
         try {
             unvalidated_pc_loc = parseSpace(command[0]);
             unvalidated_end_space = parseSpace(command[2]);
         } catch (Exception a) {
+            // REVIEW : Catching specific exceptions
```

Comment:
It is best practice to capture the most specific exception that is applicable. Catching the most-generic (the highest in the exception hierarchy in java) works, but is less than ideal.